roles/matrix-synapse/templates/synapse/homeserver.yaml.j2

```diff
@@ -525,7 +525,7 @@ max_upload_size: "{{ matrix_synapse_max_upload_size_mb }}M"
 # an explicit url_preview_ip_range_blacklist of IPs that the spider is
 # denied from accessing.
 #
-url_preview_enabled: true
+url_preview_enabled: "{{ matrix_synapse_url_preview_enabled|to_json }}"
```

Comment:
Still wrapped in double-quotes?

Reply:
Wow, I am not awake this morning. Fixed for real now.